workflow/resources/datavzrd/diffexp-template.yaml

```diff
@@ -1,5 +1,5 @@
 __use_yte__: true
-max-in-memory-rows: 15000
+max-in-memory-rows: 30000
```

Comment:
I'm assuming you have tried this with a real dataset, right?

My main question would be, if this will usually cover the full table with a standard human transcriptome. It might at least be enough for `genes_representative` and `genes_aggregated`, depending on the total number of genes. Although I do get more gene identifiers from the Ensembl cdna file that we usually use as a reference:
```
❯ zgrep "^>" Homo_sapiens.GRCh38.cdna.all.fa.gz | grep -o -P "ENSG[^\.]+" | sort | uniq | wc -l
41116
```
Thus, if this does not hurt performance of datavzrd (or rather the resulting HTML table in a browser) too much, we could think about increasing this to `42000`, right?

Reply:
Was that testing on mouse? Or on human?

If it was on human, I would stick with the 30000, otherwise we could maybe test with human data and increase it accordingly. And/or make this configurable via the `config.yaml`, explaining the performance tradeoff? But we should also always be careful not to add too many configuration options, to keep the `config.yaml` manageable...

Reply:
OK, let's go with 42000 for now, and make this configurable if we run into trouble with any reports not properly loading.